styles.css

```diff
@@ -346,3 +346,107 @@ button:hover {
   background: #6070ff;
   color: #fff;
 }
+
+button:active {
+  background-color: #2230d2;
+  color: white;
+}
+
+/*
+---------------------------------
+Contact form section begins here
+---------------------------------
+*/
+
+.contact-wrapper {
+  width: 100%;
+  height: 779px;
+  background: #6070ff;
+  z-index: 1;
+  margin-top: -100px;
+  border-radius: 100px 0 0 0;
+  background-image: url('./images/contact-mob-bg.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position-x: 153px;
+}
+
```

Comment:
- Your Design: 
![image](https://github.com/cloud007bond/Portfolio/assets/60083437/dcd13871-ca5d-465d-9386-7d3a0d049155)

- Given Design:
![image](https://github.com/cloud007bond/Portfolio/assets/60083437/501e3f60-215a-4b4a-86bf-eef25156e465)

- [ ] Kindly compare both design and fix the position of the icons in background images to match the given design.